### ISSUE TITLE
MegaSync - Icons disapearing after synchronization

### ISSUE THREAD

**Issue by Gabarito007** [meganz/MEGAsync#75]
Very good to know there is a place to exchange ideas over MegaSync.

My system, Windows Seven SP1 on a Intel i7, is showing a very weird icon for files after synchronization. It simply vanishes (sample capture attached). Before, it put a small ticked mark on each original icon file, but now it disapeared at all. How to fix that?

P.S.: Something gone wrong with attaching file.
Sample pics are here:
https://s11.postimg.org/ikmm5mwkj/Icons.png
https://s21.postimg.org/7abci88fb/icons2.png

**Comment by Gabarito007** [comment]
I could find out that situation described above only occur if Mega isn't running.
When I start MegaSync.exe and refresh Explorer view, all icons are restored.

**Comment by Gabarito007** [comment]
Talking to people at GPSoftware, thay said that vanishing icons with some shell extensions were tracked down to a combination of bugs in both the shell extensions and Windows itself, where the extensions would ask for an icon to be drawn **with some invalid attributes**, and Windows would respond by drawing nothing at all.

So, it means that the problem could be on Mega Shell Extension.

Is there some work on that issue?

**Comment by Gabarito007** [comment]
I opened this issue on march, 06 but never got an answer.

On may or june, that problem vanished and all icons were behaving well.
I assigned that solution to some new version of MegaSync.
And life tracked its way.

Last week, I had to restore all system files and settings.
And that problem came back again after installing last MegaSync version from mega.nz
site.
I noticed that before system restore, my MegaSync version was slight different from the new one. Unfortunately, I don't know which one it was.

I uninstalled it and re-installed again, but the problem remains.
When MegaSync is running, all icons shows fine.
If MegaSync isn't running, I get those invisible icons.


Someone could suggest what to do?

**Comment by polmr** [comment]
First, apologies for the late response to all the stuff mentioned in this issue (somehow it got lost in space).
Second: 
```
I could find out that situation described above only occur if Mega isn't running.
```
MEGAsync needs to be running so that the file manager knows the status of the files and hence draw the overlay icons (a mini icon indicating the synchronization status of a file on the bottom-right corner of the icon). If MEGAsync is not running the raw icon should be drawn with no overlay icon on top. I cannot reproduce your issue with a Windows 7 SP1 file browser, and I don't think anyone has reported that issue.

```
Firefox browser is unabled to attach files to this Github issues page.
I tried several times with different files types without success.
First time I tried using Chrome, it worked perfectly.
```
Sorry to hear that, but I'm afraid that's a Github issue and there's nothing we can do about it. If the problem persist, please contact github support team to report that.

```
When MegaSync is running, all icons shows fine.
If MegaSync isn't running, I get those invisible icons.
```
Does refreshing (pressing Ctrl+R) solves that issue? Do you have any extra piece of software installed to tune your file browser (screenshots does not show the default expected appearance). If so, please, tell us what, so we can try to reproduce and see if there's something faulty in MEGAsync shell extension.

Also, in MEGAsync Preferences -> Advanced, you can "Disable overlay icons". You'd lose the status, but perhaps your icons will be back to normal.

**Comment by Gabarito007** [comment]
Hi, polmr.

Thank you for your attention.

I suspect that this issue is unusual, because I searched for that
previously and found no record.
But I have news: such problem had gone.
And I don't know what I did to drive the system for solution.
I remember to install on late days the following:

   - Visual C++ Redistributable for Visual Studio 2012 Update 4 -
   vcredist_x64
   - python-3.6.1-amd64-webinstall
   - VapourSynth-R37

Finally, I'd like to know what is the latest version available.
Here, I'm running version 3.5 - 5003b1.

​As I said, before I reinstall my whole system, MEGASync version was
slightly different.​
I can't say what version is was, but I liked that.

But if 3.5 was the latest one and recommended from MEGA, no reason to
refuse that.

If you know anything more to add about all this stuff, I'd appreciate it.
But the problem is gone.

​Thank you again.​



2017-11-10 7:14 GMT-02:00 polmr <notifications@github.com>:

**Comment by Gabarito007** [comment]
Hi, polmr.

Thank you for your attention.

I suspect that this issue is unusual, because I searched for that previously and found no record.
But I have news: such problem had gone.
And I don't know what I did to drive the system for solution.
I remember to install on late days the following:

    Visual C++ Redistributable for Visual Studio 2012 Update 4 - vcredist_x64
    python-3.6.1-amd64-webinstall
    VapourSynth-R37

Finally, I'd like to know what is the latest version available.
Here, I'm running version 3.5 - 5003b1.

​As I said, before I reinstall my whole system, MEGASync version was slightly different.​
I can't say what version is was, but I liked that.

But if 3.5 was the latest one and recommended from MEGA, no reason to refuse that.

If you know anything more to add about all this stuff, I'd appreciate it.
But the problem is gone.

​Thank you again.​
